docs/sources/flow/reference/components/prometheus.scrape.md

```diff
@@ -257,6 +257,19 @@ http://blackbox-exporter:9115/probe?target=grafana.com&module=http_2xx
 http://blackbox-exporter:9116/probe?target=grafana.com&module=http_2xx
 ```
 
-## Compression
+### Technical details
 
 `prometheus.scrape` supports [gzip](https://en.wikipedia.org/wiki/Gzip) compression.
+
+The following special labels can change the behavior of prometheus.scrape:
+* `__address__` is the name of the label that holds the address of a scrape target.
```

Comment:
```suggestion
* `__address__` is the name of the label that holds the `<host>:<port>`  address of a scrape target.
```

WDYT? It'd tripped me up in the past, may be helpful here.

Reply:
Making it clear exactly what address... a good idea. (side thought: Are there other areas where being more explicit like thsi will help?)

There's an extra space between port and address.